Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -529,8 +526,10 @@ void ViewTransition::activateViewTransition()
     }
 
     for (auto& [name, capturedElement] : m_namedElements.map()) {
-        if (RefPtr newElement = capturedElement->newElement.get())
-            newElement->setCapturedInViewTransition(true);
+        if (auto newStyleable = capturedElement->newElement.styleable()) {
+            if (newStyleable->renderer())
+                newStyleable->renderer()->setCapturedInViewTransition(true);
```

Comment:
```suggestion
            if (CheckedPtr renderer = newStyleable->renderer())
                renderer->setCapturedInViewTransition(true);
```

`Styleable::renderer()` is not free, we should cache the result if we can instead of computing it twice